torch_glow/src/PyTorchModelLoader.cpp

```diff
@@ -4334,6 +4335,46 @@ Error PyTorchModelLoader::loadTranspose(const torch::jit::Node *ptNode) {
   RETURN_ERR(addValueMapping(outputs[0], output->getResult(), dtype));
 }
 
+Error PyTorchModelLoader::loadCopy(const torch::jit::Node *ptNode) {
+  auto inputs = ptNode->inputs();
+  auto outputs = ptNode->outputs();
+
+  RETURN_IF_ERR(checkInputAndOutputSizes(inputs, 3, outputs, 1));
+
+  glow::NodeValue lhs, rhs;
+  ASSIGN_VALUE_OR_RETURN_ERR(lhs, getGlowNodeValueForValue(inputs[0]));
```

Comment:
Would you mind giving every input an explicit name? Like CopyInput::lhs = 0, etc. You can reference from other ops.

Reply:
Done.